data/data/azure/cluster/main.tf

```diff
@@ -52,6 +52,7 @@ module "master" {
   private                    = var.azure_private
   outbound_udr               = var.azure_outbound_user_defined_routing
   ultra_ssd_enabled          = var.azure_control_plane_ultra_ssd_enabled
+  accelerated_networking     = var.azure_control_plane_accelerated_networking_enabled
```

Comment:
Can we be consistent in the tf files with the name that we use? We have `accelerated_networking_enabled` as the input variable from the installer, `accelerated_networking` as the input variable to the master module, and `enable_accelerated_networking` as the field of the azurerm_network_interface resource.